include/ignition/sensors/AirPressureSensor.hh

```diff
@@ -67,7 +67,8 @@ namespace ignition
       /// \brief Update the sensor and generate data
       /// \param[in] _now The current time
       /// \return true if the update was successfull
-      public: virtual bool Update(const common::Time &_now) override;
+      public: virtual bool Update(
+        const std::chrono::steady_clock::duration &_now) override;
```

Comment:
All these headers are being installed, so we can't just change the API, we need to go through a deprecation cycle.